README.adoc

```diff
@@ -1,4 +1,4 @@
-# Red Hat Single Sign-On 7.3 continuous delivery (CD) Tech Preview OpenShift container image
+# Red Hat Single Sign-On continuous delivery (CD) Tech Preview OpenShift container image
```

Comment:
Isn't this missing number 6 somewhere?

Reply:
I'm trying to remove all this unnecessary references to a specific version. I guess that one should be fine to be removed. 

BTW, I will change `continuous delivery` to `Continuous Delivery` in a minute.